services/core-web/src/components/noticeOfWork/applications/NOWSubmissionDocuments.js

```diff
@@ -64,33 +105,167 @@ export const NOWSubmissionDocuments = (props) => {
     },
   ];
 
-  const columns = [fileNameColum, ...otherColumns];
+  if (!props.hideImportStatusColumn) {
```

Comment:
Is it possible to add an additional column to this table to show its the document in the final application package or not? Similar to the other document tables. (Not a requirement for merging this or, more of curiosity if we have that data available here)

Reply:
Hm, interesting, but then the table may look too overloaded and be a duplication because we already have a separate Final Application Package on the Administrative tab with that information